server/src/socket/handlers/handleConnectedUser.js

```diff
@@ -0,0 +1,42 @@
+import { connectedUserAckEvent, userOnlineEvent, exceptionEvent } from '../../constants/eventNames';
```

Comment:
are you guys not using eslint? or husky?

Reply:
we are using, it seems that this line has not yet crossed max-length.
I will check it again though